docs/editors-guide/externally-managed-content.md

```diff
@@ -0,0 +1,33 @@
+## Automatic Mondo Quality Control Checks for Externally Managed Content
+A subset of the extensive Mondo QC checks are automatically run on externally managed content. Any content from external providers that does not pass these QC checks is not included in Mondo until the content is updated and passes the QC checks. Reports of these QC checks will be available for review of any errors so that any needed changes can be made.
+
+### QC Checks for Externally Managed Content
+| Error | Description | SPARQL Query | 
+|:------|:------------|:-------------|
+| duplicate_scoped_synonym | An entity has duplicate synonyms with different properties (e.g. the same broad and related synonym). This causes ambiguity. | <a href='https://robot.obolibrary.org/report_queries/duplicate_scoped_synonym' target='_blank'>duplicate scoped synonym</a> |
+| invalid_xref | A database_cross_reference is not in CURIE format. | <a href='https://robot.obolibrary.org/report_queries/invalid_xref' target='_blank'>invalid xref</a> |
+| qc-duplicate-exact-synonym-no-abbrev| An exact synonym, excluding abbreviations, or label is used for two different entities.| <a href='https://github.com/monarch-initiative/mondo/blob/master/src/sparql/qc/general/qc-duplicate-exact-synonym-no-abbrev.sparql' target='_blank'>duplicate synonyms</a>|
+| qc-related-exact-synonym| A value for a synonym is used with both the exact and related synonym scopes.|<a href='https://github.com/monarch-initiative/mondo/blob/master/src/sparql/qc/general/qc-related-exact-synonym.sparql' target='_blank'>related exact synonym</a>|
+| qc-syn-source-not-xref| A synonym does not have a database cross reference as the source of the synonym.| <a href='https://github.com/monarch-initiative/mondo/blob/master/src/sparql/qc/general/qc-syn-source-not-xref.sparql' target='_blank'>missing synonym source</a>|
+| qc-trailing-whitespace.sparql| An entity contains a leading or trailing whitespace. | <a href='https://github.com/monarch-initiative/mondo/blob/master/src/sparql/qc/general/qc-trailing-whitespace.sparql' target='_blank'>trailing whitespace</a>|
```

Comment:
Is this a EMC related QC check? I am assuming that we have a check to make sure that we remove the extra whitespace when we process these files. 

Do we really exclude information and tell the source to remove trailing whitespace? (that seems a bit weird to me)

Reply:
well, this is an existing QC check for Mondo so if the trailing whitespace is not fixed at the source each month a curator will need to fix this manually. this is added because it has happened in past additions of EMC content

Reply:
however, i also agree that removing any leading/trailing whitespace should be something that happens when processing the EMC content